.hype/actions/hype.yml

```diff
@@ -2,30 +2,29 @@ name: Generate README with Hype
 on: [pull_request]
 
 jobs:
-
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4
-      with:
-        repository: ${{ github.event.pull_request.head.repo.full_name }}
-        ref: ${{ github.event.pull_request.head.ref }}
-    - name: Set up Go
-      uses: actions/setup-go@v4
-      with:
-        go-version: "1.22.x"
-        cache-dependency-path: subdir/go.sum
+      - uses: actions/checkout@v4
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: "1.22.x"
+          cache-dependency-path: subdir/go.sum
 
-    - name: Install hype
-      run: go install github.com/gopherguides/hype/cmd/hype@latest
+      - name: Install hype
+        run: go install github.com/gopherguides/hype/cmd/hype@latest
 
-    - name: Run hype
-      run: pushd .hype;hype export -format=markdown -f module.md > ../README.md;popd
+      - name: Run hype
+        run: pushd .hype;hype export -format=markdown -f hype.md > ../README.md;popd
```

Comment:
per the previous comment, we could remove this file then and really show this is about re-using content and never getting your content out of date.